pxtrunner/runner.ts

```diff
@@ -363,7 +363,7 @@ namespace pxt.runner {
             })
     }
 
-    export async function simulateAsync(container: HTMLElement, simOptions: SimulateOptions): Promise<pxtc.BuiltSimJsInfo> {
+    export async function simulateAsync(container: HTMLElement, simOptions: SimulateOptions): Promise<pxsim.SimulatorDriver> {
```

Comment:
we actually use the value returned here in arcades share page for perf when toggling between editor view & full screen game -- so ideally we should either not modify or return an object with both values available